packages/noya-state/src/reducers/application.ts

```diff
@@ -121,7 +121,8 @@ export type Action =
       mode?: SetNumberMode,
     ]
   | [type: 'addColorSwatch']
-  | [type: 'addLayerStyle']
+  | [type: 'addLayerStyle', name?: string, style?: Sketch.Style]
+  | [type: 'updateLayerStyle', id: string, style: Sketch.Style | undefined]
```

Comment:
For each of the new actions, can we clarify `id` by making it either `layerId` or `sharedStyleId`?